guide/english/machine-learning/index.md

```diff
@@ -55,6 +57,7 @@ data consist of a set of training examples. In supervised learning, each example
 3. If you want a brief introduction of machine learning, and you prefer videos, try this <a href='https://youtu.be/cKxRvEZd3Mw' target='_blank' rel='nofollow'>machine learning introduction video</a>
 4. If you want to know how to proceed with learning machine learning, take a look at this <a href='https://youtu.be/nKW8Ndu7Mjw' target='_blank' rel='nofollow'> video</a>
 5.<a href='https://www.datacamp.com/courses/supervised-learning-with-scikit-learn' target='_blank' rel='nofollow'>Supervised Learning on Data camp</a>
+6.<a href='https://machinelearningmastery.com/' target='_blank' rel='nofollow'>Machine Learning Mastery</a> 
```

Comment:
Add spacing between bullet points & text for 5. and 6. to fix markdown formatting.

e.g. change:

```md
5.<a href='...
6.<a href='...
```

to 

```md
5. <a href='...
6. <a href='...
```